packages/circuit-ui/components/CalendarTag/CalendarTag.tsx

```diff
@@ -93,16 +115,13 @@ class CalendarTag extends Component {
 
     return (
       <div {...props}>
-        <Tag
-          selected={isOpen}
-          ref={this.handleTagRef}
-          onClick={this.handleTagClick}
-        >
+        <Tag selected={isOpen} ref={this.tagRef} onClick={this.handleTagClick}>
           {this.getDateRangePreview()}
         </Tag>
         {isOpen && (
           <CalendarWrap>
             <RangePickerController
+              initialVisibleMonth={null}
```

Comment:
Added this to make TypeScript happy. The component works the same with and without 🤷‍♂️